README.md

```diff
@@ -77,8 +23,9 @@ Yorkie CodePair is deployed to AWS and the configuration is shown below. This re
 
 ```
 [Route53]
- ㄴ codepair.yorkie.dev - [gh-pages] # for serving static pages
- ㄴ api.yorkie.dev - [ELB] - [EC2]   # for serving API
+ ㄴ codepair.yorkie.dev - '/'    [gh-pages]   # for serving static pages
+                       ㄴ '/api' [api-server] # for serving API (not yet)
+ ㄴ api.yorkie.dev - [ELB] - [EC2]            # for serving yorkie API
```

Comment:
In Route53, it is difficult to branch based on paths like L7, so it would be better to add a separate domain.

```suggestion
 ㄴ codepair.yorkie.dev - [gh-pages]   # for serving static pages
 ㄴ codepair-api.yorkie.dev [api-server] # for serving API (not yet)
 ㄴ api.yorkie.dev - [ELB] - [EC2]            # for serving yorkie API
```